docs/services/development-services/development-services.sass

```diff
@@ -1358,6 +1590,123 @@ $step6: 1919px
+
+            .sub-title
+                font-size: 30px
+                font-weight: 600
+                line-height: 44px
+                letter-spacing: 0.01em
+                text-align: center
+                color: #333333
+                display: flex
+                justify-content: center
+
+            .sub-sub-title
+                font-size: 16px
+                font-weight: 400
+                line-height: 28px
+                letter-spacing: 0.001em
+                text-align: center
+                color: #5E5F62
+                display: flex
+                justify-content: center
+                margin-top: 15px
+                span    
+                    width: 556px
+                    height: 56px
+
+            .form-section
+                padding-top: 30px
+                padding-left: 40px
+                padding-right: 40px
+                display: flex
+                justify-content: space-around
+                .form-element
+                    margin-bottom: 15px
+                    width: 48%
+                    label
+                        position: relative
+                        width: 100%
+                        p
+                            color: rgba(0, 0, 0, 0.54)
+                            font-size: 12px
+                            font-style: normal
+                            font-weight: 400
+                            line-height: 16px
+                            letter-spacing: 0.4px
+                            position: absolute
+                            top: -8px
+                            left: 10px
+                            z-index: 2
+                            padding: 0px 5px
+                            background: white
+                    
+                        select
+                            width: 100%
+                            appearance: none
+                            background-image: url(/images/development-services/arrow-down.svg) 
+                            background-repeat: no-repeat
+                            background-position: right 16px center
+                            color: rgba(0,0,0,0.54)
+                        .cdu-form-control
+                            width: 100%
+                            font-size: 16px
+                            line-height: 24px
+                            letter-spacing: 0.15px
+                            border-radius: 4px
+                            border: 1px solid rgba(0, 0, 0, 0.12)
+                            padding: 16px 18px
+
+                        .cdu-text-area
+                            height: 170px
+
+            .submit-button-container
+                display: flex
+                justify-content: center
+                .cdu-button
+                    padding: 15px
+                    border-radius: 6px
+                    color: white
+                    background-color: #2A7DEC
+                    width: 400px
```

Comment:
It is possible to rewrite the styles in a more simplified version:
```
$modal-z-index: 13000;
$modal-background-color: rgba(0, 0, 0, 0.4);
$modal-content-background-color: #fefefe;
$modal-content-border: 1px solid #888;
$title-font: 16px/30px; // abbreviations for font-size and line-height
$sub-title-font: 30px/44px;
$sub-sub-title-font: 16px/28px;
$form-element-label-color: rgba(0, 0, 0, 0.54);
$form-control-border: 1px solid rgba(0, 0, 0, 0.12);
$button-background-color: #2A7DEC;
$font-style-normal: normal;

.modal {
    display: none;
    position: fixed;
    z-index: $modal-z-index;
    left: 0;
    top: 0;
    width: 100%;
    height: 100%;
    background-color: $modal-background-color;

    &-content {
        background-color: $modal-content-background-color;
        padding: 20px 20px 30px; // скорочена форма padding
        border: $modal-content-border;
        width: 950px;

        .close-button {
            display: flex;
            justify-content: flex-end;
        }

        .close {
            width: 28px;
            height: 28px;
            cursor: pointer;
        }

        .title,
        .sub-title,
        .sub-sub-title {
            font-weight: 600;
            letter-spacing: 0.01em;
            text-align: center;
            color: #3622A4; // для .title
            display: flex;
            justify-content: center;
        }

        .title { font: $title-font $font-style-normal; color: #3622A4; }
        .sub-title { font: $sub-title-font $font-style-normal; color: #333333; }
        .sub-sub-title { font: $sub-sub-title-font 400 $font-style-normal; color: #5E5F62; margin-top: 15px; }

        .form-section {
            padding: 30px 40px;
            display: flex;
            justify-content: space-around;

            .form-element {
                margin-bottom: 15px;
                width: 48%;

                label {
                    position: relative;
                    width: 100%;
                    p {
                        color: $form-element-label-color;
                        font: 12px/16px $font-style-normal 400;
                        letter-spacing: 0.4px;
                        position: absolute;
                        top: -8px;
                        left: 10px;
                        z-index: 2;
                        background: white;
                    }

                    .cdu-form-control,
                    .cdu-text-area {
                        width: 100%;
                        color: #898B8F;
                        font: 16px/24px $font-style-normal 400;
                        letter-spacing: 0.15px;
                        border-radius: 4px;
                        border: $form-control-border;
                        padding: 16px 18px;
                    }

                    .cdu-text-area { height: 170px; }
                }
            }
        }

        .submit-button-container {
            display: flex;
            justify-content: center;

            .cdu-button {
                padding: 15px;
                border-radius: 6px;
                color: white;
                background-color: $button-background-color;
                width: 400px;
            }
        }
    }
}
```

Reply:
Fixed